app-modules/meeting-center/src/Models/Event.php

```diff
@@ -71,4 +76,39 @@ public function attendees(): HasMany
     {
         return $this->hasMany(EventAttendee::class, 'event_id');
     }
+
+    public static function executeFromCampaignAction(CampaignAction $action): bool|string
+    {
+        if (app(LicenseSettings::class)->data->addons->eventManagement) {
```

Comment:
I think this is the first time that we have a campaign action that is gated by a feature/addon.

What happens if this is attempted to be executed when the addon is turned off? Does it retry to run indefinitely?